Add some output to the --nuke command

PromoFaux opened the pull request:
**By submitting this pull request, I confirm the following:** 

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**

Addresses #3845

output after PR:

![image](https://user-images.githubusercontent.com/1998970/99314652-56e54b00-2859-11eb-99ce-31be6bf6314c.png)

DL6ER reviewed (commented):
Shouldn't it be

> Removed ${count} domain(s) from **the** ${listname}

?

Also, why printing the number two times? Maybe

> Removed **all** domains from ...

PromoFaux commented:
>**the** ${listname}

I agree with that, oversight on my part.

What I was going for with the latter was to say "there are `n` number of domains in total on the ${listname}, and now I have removed `n` domains", rather than just "I removed some domains" I think it would probably make more sense with some if conditions in there and a check after the delete command has run.. but likely overkill

PromoFaux commented:
Force pushed some changes

![image](https://user-images.githubusercontent.com/1998970/99320647-00c9d500-2864-11eb-825f-7876546b75e0.png)
